### DISCUSSION TITLE
Tiles3DLoader renderer error

### DISCUSSION THREAD

**Discussion by Lshanny** [visgl/deck.gl#6394] [Q&A]
When I use tiles3dloader to load the point cloud 3dtiles file converted by cesiumlab, an error occurs：
![image](https://user-images.githubusercontent.com/39947011/141750740-5cc77f3b-fbf1-45e3-a230-25f882dd1c9f.png)
new Tile3DLayer({
      id: 'tile-3d-layer',
      data:'/data/dy2/tileset.json',  //my data url
      loadOptions: {
        '3d-tiles':{isTileset:true},
        isTileset:true
      }
})
Please give me more advice on how to solve it

**Comment by Pessimistress** [comment]
If you could share your dataset, folks over at https://github.com/visgl/loaders.gl can help you debug.
